src/io/flutter/inspector/FlutterStageView.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.flutter.inspector;
+
+import com.android.tools.adtui.AxisComponent;
+import com.android.tools.adtui.model.AspectObserver;
+import com.android.tools.adtui.model.Range;
+import com.android.tools.adtui.model.formatter.TimeFormatter;
+import com.android.tools.profilers.*;
+import com.intellij.ui.components.JBPanel;
+import icons.StudioIcons;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.util.concurrent.TimeUnit;
+
+import static com.android.tools.profilers.ProfilerFonts.STANDARD_FONT;
+
+public abstract class FlutterStageView<T extends FlutterStage> extends AspectObserver {
+  private final T myStage;
```

Comment:
here and elsewhere feel free to omit the "my" prefix for field names. Generally the intellij flutter plugin isn't consistent on this.